docs/contracts/v4/guides/02-manage-liquidity/04-collect.mdx

```diff
@@ -1,3 +1,66 @@
 ---
 title: Collect Fees
----
\ No newline at end of file
+---
+
+### Setup
+
+See the [setup guide](./00-setup-liquidity.mdx)
+
+# Guide
+
+In order to collect fees, the integrator must execute encoded actions
+using the `PositionManager` contract.
+
```

Comment:
lets add a note here, explaining that there is not `COLLECT` command. instead, developers should decrease liquidity with a zero liquidity change